cmd/all-in-one/all_in_one_test.go

```diff
@@ -166,7 +166,7 @@ func getAPITrace(t *testing.T) {
 	}
 
 	require.Len(t, queryResponse.Data, 1)
-	require.Len(t, queryResponse.Data[0].Spans, 1)
+	require.Len(t, queryResponse.Data[0].Spans, 2)
```

Comment:
Here are the spans generated before this change
<img width="1462" alt="Screenshot 2024-10-22 at 9 51 34 PM" src="https://github.com/user-attachments/assets/6105c1ab-1816-4c91-90f2-1ee7517f0f47">

and after 
![image](https://github.com/user-attachments/assets/57c44176-d324-4e5f-bf8a-33168d24a847)